tests/Sdk/TransactionApiTest.php

```diff
@@ -111,7 +111,7 @@ public function testGetByBlockNum(): void
 
         $transaction = $this->sdk->getTransactionApi()->getByBlockNum(1370282, 0);
         $this->assertEquals('1f140e5744bcef282147ef3f0bab8df46f49704a99046d6ea5db37ab3113e0f45935fd94af7b33189ad34fa1666ab7e54aa127d725e2018fb6b68771aacef54c41', $transaction->getSignatures()[0]);
-        $this->assertEquals('2018', $transaction->getExpiration());
+        $this->assertEquals(59561, $transaction->getRefBlockNum());
```

Comment:
Isn't this value going to change over time? I guess test against mock server will work, but integration test will stop working over time. Please verify.

Reply:
Fixed by changing the assert.